scripts/test-theme.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/zsh
```

Comment:
This is just a script to make testing the code easier. It copies `theme.json` to `target/assets/theme-customer.json` and adds it as a theme to `config.json`. `jq` needs to be installed for this to work.
`yarn build && ./scripts/test-theme.sh` if you want to the run the PR.